### DISCUSSION TITLE
Flutter web billing address The export transaction required customer name and address

### DISCUSSION THREAD

**Discussion by ankitmiyan30** [flutter-stripe/flutter_stripe#1521] [Q&A]
Can someone help with this how to get through this error?
I am using an Indian merchant account to take payments from customers in international currency and it's giving me this error with both Cardfield and PaymentElement.

Getting the same error in both CardField and PaymentElement for Flutter Web.
The export transaction required customer customer name and address.
<img width="569" alt="Screenshot 2023-12-04 at 4 46 28 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/30353044/b2f8fde9-84cf-4c40-b7ea-4639e1a9470a">
<img width="1122" alt="Screenshot 2023-12-04 at 4 50 02 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/30353044/ee99a087-d703-4802-b266-b3a67a44ec24">
<img width="415" alt="Screenshot 2023-12-04 at 5 45 18 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/30353044/93a9da1d-37a2-48a1-a6de-b5a97926ad84">
<img width="1063" alt="paymentele" src="https://github.com/flutter-stripe/flutter_stripe/assets/30353044/6c7a3123-fe35-47b9-bb3b-a0725d55f0a5">

**Comment by remonh87** [comment]
You need to supply the data to your payment intent. Our billing details are only used to prepopulate data in the payment element.

**Reply by ankitmiyan30** [reply]
@remonh87 create payment intent doesn't allow me to send the address along with the request could you please tell me how to send the billing address object with create payment inter api?
this is how i am calling create payment intent api

Map<String, dynamic> body = {
        'amount': price,
        'currency':'USD',
        'capture_method': 'manual',
        'description': desc,
      };
      log(latestRoomPrice.value.roomGroup![0].totalRate!.toString());
      log((latestRoomPrice.value.roomGroup![0].totalRate! * 100).toString());

      var response = await http.post(
        Uri.parse('${stripeBaseUrl}payment_intents'),
        headers: {'Authorization': stripAuthKey, 'Content-Type': 'application/x-www-form-urlencoded'},
        body: body,
      );

**Reply by remonh87** [reply]
Normally you would create a customer: https://stripe.com/docs/api/customers and then you link the paymentintent tot the customer. But you can also contact stripe support how best facilitate your use case.

**Reply by ankitmiyan30** [reply]
@remonh87 I am able to create customers but how can we make the relation between a created customer and a created payment intent, could you help me with this?

**Comment by pooja96km** [comment]
Is this resolved?

**Reply by ankitmiyan30** [reply]
No, but by calling create customer api and then passing customer id to the payment intent can help you.
